src/components/KCard/KCard.vue

```diff
@@ -1,18 +1,80 @@
+<script lang="ts" setup>
+import { v1 as uuidv1 } from 'uuid'
+import type { BorderVariant } from '@/types'
+import { computed, useSlots } from 'vue'
+
+interface IProps {
+  /**
+   * Title string if slot not used, also used for aria-label
+   */
+  title?: string;
+  /**
+     * Pass body string in if slot not used
+     */
+  body?: string;
+  /**
+     * Set top border or no border. If neither set default will have border<br>
+     * * Options: [borderTop, noBorder]
+     */
+  borderVariant?: BorderVariant;
+  /**
+   * Sets if card has hover state<br>
+   */
+  hasHover?: boolean;
+  hasShadow?: boolean;
+  /**
+   * Add small status text above the card title
+   */
+  status?: string;
+  /**
+   * Test mode - for testing only, strips out generated ids
+   */
+  testMode?: boolean;
+}
+
+const props = withDefaults(defineProps<IProps>(), {
+  title: '',
+  body: '',
+  borderVariant: 'border',
+  hasHover: false,
+  hasShadow: false,
+  status: '',
+  testMode: false,
+})
```

Comment:
We use the standard `defineProps` syntax -- can you please look at the other components and follow that convention?

Reply:
Covering over Slack

Reply:
fixed